### PR TITLE
OVA-1.0.3 Release

### PR DESCRIPTION
**Pull request by mukul-seagate11**
Update Wiki for OVA-1.0.3 Release

### CODE REVIEW

File: doc/CORTX_on_Open_Virtual_Appliance.rst [hunk 1]
```diff
@@ -20,7 +19,7 @@ Procedure
 **********
 The procedure to install CORTX on OVA is mentioned below.
 
-#. Download and unzip the `cortx-va-1.0.2.zip <https://github.com/Seagate/cortx/releases/download/VA/cortx-va-1.0.2.zip>`_ file from `our release page <https://github.com/Seagate/cortx/releases/tag/VA>`_. This contains the virtual machine image.
+#. Download `cortx-va-1.0.3.ova <https://github.com/Seagate/cortx/releases/download/ova-1.0.3/cortx-va-1.0.3.ova>`_ file from `our release page <https://github.com/Seagate/cortx/releases/latest>`_. This contains the virtual machine image.
```
Comment (shailesh-vaidya) [inline doc/CORTX_on_Open_Virtual_Appliance.rst:22]: Previously @johnbent mentioned to keep all our Releases as pre-release and add the text mentioned in https://github.com/Seagate/cortx/releases/tag/VA
Can we update https://github.com/Seagate/cortx/releases/tag/ova-1.0.3 accordingly or if are planning to publish it as Prod Release.

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]
Please remove doc.sig file . Not sure if it's required.

File: doc/CORTX_on_Open_Virtual_Appliance.rst (hunk 1 above)
Comment (mukul-seagate11) [reply doc/CORTX_on_Open_Virtual_Appliance.rst:22]: Note sure about doc.sig as its coming from https://github.com/Seagate/cortx/blob/1031200af6e0a947bec09aa028077bb1b31662f2/doc.sig commit

Updated https://github.com/Seagate/cortx/releases/tag/ova-1.0.3 with text and pre-release for now

**Comment by mukul-seagate11** [comment]
blocker due to doc.sig & DCO so will be raising a new PR

**Pull request closed without merging**